### DISCUSSION TITLE
Chest is to low underground to be protected.

### DISCUSSION THREAD

**Discussion by JBSTheGamer** [GriefPrevention/GriefPrevention#740] [Q&A] [answered]
So i wanted to know how do i make it not say that no matter how low underground you are. Because no matter how low i go underground it will still say. Chest is too low underground to be protected. I even tried to place a chest at Y=1 and it still said chest is too low underground to be protected. No matter how high i go into the sky it still says it too. I will be at Y=255 and i try to place a chest and it says it's too low underground to be protected. I know there is something wrong with the config. I just can't figure out what it is that is causing this plugin to say that your chest is to low underground to be protected. Even though i literally place the chest in the sky. So here is the config file below so you can figure out what is causing this to happen.
[config.zip](https://github.com/TechFortress/GriefPrevention/files/4289656/config.zip)

**Comment by Jikoo** [comment, marked as answer]
I didn't check your config because I'm lazy and didn't want to go through the hassle of downloading and unzipping it for a look, but I can almost guarantee you have configured `GriefPrevention.Claims.MaximumDepth` way too high.

**Comment by JBSTheGamer** [comment]
yes i saw i fixed it by debugging thanks for the help though.
